library/src/androidTest/java/com/owncloud/android/AbstractIT.java

```diff
@@ -277,4 +281,10 @@ public static File getFile(String filename) throws IOException {
 
         return temp;
     }
+
+    public static void onlyOnMin(OwnCloudVersion version) {
```

Comment:
suggestion: rename to `assumeServerVersion`, `requireServerVersion` or something like that. `onlyOnMin` is not immediately obvious